chain/client/src/info.rs

```diff
@@ -444,13 +443,14 @@ impl InfoHelper {
             })
             .unwrap_or_default();
 
-        info!(
-            target: "stats", "{}{}{}{}{}",
+        tracing::info!(
+            target: "stats",
             sync_status_log,
             validator_info_log,
             network_info_log,
             blocks_info_log,
             machine_info_log,
+            "node status"
```

Comment:
This will change the output format of the main node stats. Can you please post a before/after and make sure the change is desired? (or just build a string and pass it as msg)

Reply:
Before:
```
 0.090s  INFO check_triggers:log_summary: stats: #       1 2dQ7S7zwtKDVkukVofoC3x5nncnjf6Fs1kqUNHKJe4nb Validator | 2 validators 0 peers ⬇ 0 B/s ⬆ 0 B/s 0.00 bps 0 gas/s CPU: 0%, Mem: 41.5 MB
```
After:
```
 0.075s  INFO check_triggers:log_summary: stats: node status sync_status_log="#       1 2dQ7S7zwtKDVkukVofoC3x5nncnjf6Fs1kqUNHKJe4nb" validator_info_log=" Validator | 2 validators" network_info_log=" 0 peers ⬇ 0 B/s ⬆ 0 B/s" blocks_info_log=" 0.00 bps 0 gas/s" machine_info_log=" CPU: 0%, Mem: 41.3 MB"
```

In case this doesn't look good, I can revert back

Reply:
I'm reverting back to the original format except for a tiny change where we would see the log line as `node_status="..."`